packages/landingpage-new/components/home/core-aspects/core-aspects.module.scss

```diff
@@ -5,13 +5,60 @@
   width: 100%;
   background-color: #e9e5fe;
   border-radius: 267px 0px;
+  column-gap: 40px;
 
   div {
-    padding-top: 122px;
+    height: 350px;
     display: flex;
     flex-direction: column;
     align-items: center;
+    justify-content: start;
```

Comment:
The `start` value has a mixed browser support. We should use `flex-start` instead